src/lib/FreeC/Backend/Coq/Converter/Arg.hs

```diff
@@ -27,12 +27,15 @@ import           FreeC.Monad.Converter
 convertTypeVarDecls
   :: Coq.Explicitness -- ^ Whether to generate an explicit or implicit binder.
   -> [IR.TypeVarDecl] -- ^ The type variable declarations.
-  -> Converter [Coq.Binder]
+  -> Converter ([Coq.Binder], [Coq.Qualid])
```

Comment:
The meaning of the additional return value should be documented.
Alternatively, you could also just lookup the Coq identifiers in `convertFuncHead`.